docs/neo-n3/dapp_development/transactions.md

```diff
@@ -182,24 +173,23 @@ tx.track().subscribe(blockIndex -> {
 });
 ```
 
-If you call `getApplicationLog()` before the transaction is included in a block it throws an `RpcResponseErrorException`
-with an error message, that the transaction is unknown or could not be found.
+If you call `getApplicationLog()` before the transaction is included in a block, it will throw an `RpcResponseErrorException` indicating that the transaction is unknown or could not be found.
+If you call `getApplicationLog()` before the transaction is included in a block, it will throw an `RpcResponseErrorException` indicating that the transaction is unknown or could not be found.
+
+## Adding Additional Network Fees
```

Comment:
```suggestion
If you call `getApplicationLog()` before the transaction is included in a block, it will throw an `RpcResponseErrorException` indicating that the transaction is unknown or could not be found.

```